config/manager/kustomization.yaml

```diff
@@ -10,8 +10,8 @@ generatorOptions:
 
 images:
 - name: controller
-  newName: quay.io/openshift-kni/oran-o2ims-operator
-  newTag: 4.16.0
+  newName: quay.io/alegacy/oran-o2ims
```

Comment:
I don't think this needs to change.

Reply:
nope!  forgot to remove it before committing.